fastapi/security/api_key.py

```diff
@@ -116,6 +122,12 @@ class APIKeyHeader(APIKeyBase):
     """
     API key authentication using a header.
 
+    Note:
+    This class does **not** perform API key validation or decoding.
+    It only extracts the key from the request and provides OpenAPI integration.
+
+    You must implement the actual verification logic yourself (e.g., checking against a database or list of valid keys).
```

Comment:
```suggestion
        You must implement the actual verification logic yourself (e.g., checking against a database or list of valid keys).
```
Removed empty line and added indentation